src/main/java/com/northconcepts/datapipeline/foundations/examples/pipeline/ReadFromCsvWriteToExcel.java

```diff
@@ -10,55 +9,44 @@
 import com.northconcepts.datapipeline.foundations.pipeline.output.ExcelPipelineOutput;
 import com.northconcepts.datapipeline.job.Job;
 
-import java.nio.channels.Pipe;
 import java.util.LinkedHashMap;
 import java.util.Map;
 
 public class ReadFromCsvWriteToExcel {
 
-    public static void main(String[] args) throws Throwable{
+    public static void main(String[] args) {
 
         Pipeline pipeline = new Pipeline();
 
-        LocalFile inputFile = new LocalFile()
-                .setName("Input File")
-                .setPath("PATH_TO_INPUT_FILE.csv")
-                .detectFileType();
-
         CsvPipelineInput pipelineInput = new CsvPipelineInput()
-                .setFileSource(inputFile)
+                .setFileSource(new LocalFile()
+                        .setName("Input File")
```

Comment:
Let's drop `setName()` unless it's required.  Also, is the `detectFileType()` call needed?

Reply:
removed setName() and detectFileType()